presto-docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -259,6 +261,133 @@ Field           Required  Type      Description
 
 There is no limit on field descriptions for either key or message.
 
+Row encoding
+------------
+
+Encoding allows SQL insert statements to be used and defines how table columns in Presto
+map to Kafka key and message data.
+
+The Kafka connector contains the following encoders:
+
+* `CSV encoder`_ - Kafka message is formatted as comma separated message, and fields are mapped to table columns
+* `Avro encoder`_ - Kafka message is mapped to Avro fields based on an Avro schema
+
+.. note::
+
+    A table definition file must be defined for the encoder to work. `Table Definition Files`_
+
+CSV encoder
+^^^^^^^^^^^^^^^
+
+The CSV encoder formats the values for each row as a line of comma-separated-values (CSV)
+using UTF-8 encoding.
+
+The ``type`` and ``mapping`` attributes must be defined for each field:
+
+* ``type`` - Presto data type
+* ``mapping`` - The integer index of the column in the CSV line (the first column is 0, the second is 1, and so on)
+
+``dataFormat`` and ``formatHint`` are not supported and must be omitted.
+
+The following table lists supported Presto types, which can be used in ``type`` and encoding scheme:
+
+============================= ================================================
+Presto data type              Encoding rules
+============================= ================================================
+``BIGINT``                    Formatted using Java ``Long.toString()``
+``INTEGER``                   Formatted using Java ``Integer.toString()``
+``SMALLINT``                  Formatted using Java ``Short.toString()``
+``TINYINT``                   Formatted using Java ``Byte.toString()``
+``DOUBLE``                    Formatted using Java ``Double.toString()``
+``REAL``                      Formatted using Java ``Float.toString()``
+``BOOLEAN``                   Formatted using Java ``Boolean.toString()``
+``VARCHAR`` / ``VARCHAR(x)``  Unicode String encoded in UTF-8
+============================= ================================================
+
+Example CSV field definition for a Kafka message:
```

Comment:
So where do you put that file and how do you configure Presto / the connector to use it